examples/navier-stokes/index.rst

```diff
@@ -119,30 +119,32 @@ and with
 .. math::
    f(t^n, \boldsymbol{q}_N^n) = - [\nabla \cdot \boldsymbol{F}(\boldsymbol{q}_N)]^n + [S(\boldsymbol{q}_N)]^n \, .
 
-The strong form of :math:numref:`eq-vector-ns` is:
+To obtain a finite element discretization, we first multiply the strong form :math:numref:`eq-vector-ns` by a test function :math:`\boldsymbol v \in H^1(\Omega)` and integrate,
 
 .. math::
-   :label: eq-strong-vector-ns
+   \int_{\Omega} \boldsymbol v \cdot \left(\frac{\partial \boldsymbol{q}_N}{\partial t} + \nabla \cdot \boldsymbol{F}(\boldsymbol{q}_N) \right) \,dV = \int_\Omega v \mathbf{S}(\boldsymbol{q}_N) \, dV \, , \; \forall \boldsymbol v \in \mathcal{V}_p\,,
 
-   \int_{\Omega} v \left(\frac{\partial \boldsymbol{q}_N}{\partial t} + \nabla \cdot \boldsymbol{F}(\boldsymbol{q}_N) \right) \,dV = \int_\Omega v \mathbf{S}(\boldsymbol{q}_N) \, dV \, , \; \forall v \in \mathcal{V}_p
+with :math:`\mathcal{V}_p = \{ \boldsymbol v(\mathbf x) \in H^{1}(\Omega_e) \,|\, \boldsymbol v(\mathbf x_e(\mathbf X)) \in P_p(\boldsymbol{I}), e=1,\ldots,N_e \}` a mapped space of polynomials containing at least polynomials of degree :math:`p` (with or without the higher mixed terms that appear in tensor product spaces).
 
-with :math:`\mathcal{V}_p = \{ v \in H^{1}(\Omega_e) \,|\, v \in P_p(\boldsymbol{I}), e=1,\ldots,N_e \}`.
-
-And its weak form is:
+Integrating by parts, we arrive at the weak form,
 
 .. math::
    :label: eq-weak-vector-ns
 
    \begin{multline}
-    \int_{\Omega} v \frac{\partial \boldsymbol{q}_N}{\partial t}  \,dV + \int_{\partial \Omega} v \widehat{\mathbf{n}} \cdot \boldsymbol{F} (\boldsymbol{q}_N) \,dS - \int_{\Omega} \nabla v\cdot\boldsymbol{F}(\boldsymbol{q}_N)\,dV  =
-        \int_\Omega v \mathbf{S}(\boldsymbol{q}_N) \, dV \, , \; \forall v \in \mathcal{V}_p
+    \int_{\Omega} \boldsymbol v \cdot \frac{\partial \boldsymbol{q}_N}{\partial t}  \,dV + \int_{\partial \Omega} \boldsymbol v \cdot \boldsymbol{F}(\boldsymbol q_N) \cdot \widehat{\mathbf{n}} \,dS - \int_{\Omega} \nabla \boldsymbol v \!:\! \boldsymbol{F}(\boldsymbol{q}_N)\,dV  =
+        \int_\Omega \boldsymbol v \cdot \mathbf{S}(\boldsymbol{q}_N) \, dV \, , \; \forall \boldsymbol v \in \mathcal{V}_p \,,
    \end{multline}
 
-We solve equation :math:numref:`eq-weak-vector-ns` with Galerkin method discretization
-which is the default option in our Navier-Stokes example.
+where :math:`\boldsymbol{F}(\boldsymbol q_N) \cdot \widehat{\mathbf{n}}` is typically replaced with a boundary condition.
+
+.. note::
```

Comment:
Thanks @jedbrown for adding this note on the notation. I would add this also to the Theoretical Framework section where this symbol is first used.

Reply:
Do you know a clean way to display the same admonition in multiple places, or does it need to be in its own file that gets included?

Reply:
I would say it has to be in the same file where it's displayed. Also, now that I can ask you this related question. Why the need for renaming the file in your commit [10423a](https://github.com/CEED/libCEED/commit/10423a34c0f42ed3ce2e01fda83fe72572c90501). Sorry if I had missed this detail before. Thanks.

Reply:
The rename was so Sphinx doesn't complain about the file not appearing in a toctree. That's the pattern used in other projects. I adopted the naming convention used by SciPy.
https://stackoverflow.com/questions/15249340/warning-document-isnt-included-in-any-toctree-for-included-file
https://github.com/sphinx-doc/sphinx/issues/5281

Reply:
Ah! That was a good hack! I had bumped into the same thing earlier

Reply:
@LeilaGhaffari would you mind adding the same note about the ":" notation in the FEMtheory.rst.inc file, right after it is used in eq. (1)? Thanks!